doc/source/docs/user_guide/data_structures.rst

```diff
@@ -89,41 +88,43 @@ The "geometry" column -- no matter its name -- can be accessed through the :attr
 
 A :class:`GeoDataFrame` may also contain other columns with geometrical (shapely) objects, but only one column can be the active geometry at a time. To change which column is the active geometry column, use the :meth:`GeoDataFrame.set_geometry` method.
 
-An example using the ``worlds`` GeoDataFrame:
+An example using the ``geoda.malaria`` dataset from ``geodatasets`` containing the districts of Colombia:
 
 .. ipython:: python
 
-    world = geopandas.read_file(geopandas.datasets.get_path('naturalearth_lowres'))
+    import geodatasets
+
+    colombia = geopandas.read_file(geodatasets.get_path('geoda.malaria'))
```

Comment:
I would probably go for `"geoda.malaria"` because if you want to retrieve the Dataset object, you do `geodatasets.data.geoda.malaria`. We can keep it consistent with that.

Reply:
That makes sense!